gbif/ingestion/pre-backbone-release/src/main/java/org/gbif/pipelines/backbone/impact/BackbonePreRelease.java

```diff
@@ -258,27 +277,14 @@ private static String toTabDelimited(
           verbatim.getRank(), // avoid breaking the API (verbatimTaxonRank)
           verbatim.getScientificName(),
           verbatim.getGenericName(),
-          verbatim.getScientificNameAuthorship(),
+          "",
           current.toString(skipKeys),
           proposed.toString(skipKeys));
     }
 
-    @Teardown
-    public void tearDown() {
```

Comment:
no need to close it?

Reply:
i dont think there is anything to tear down when using Feign. If there is, happy to add.

Reply:
you're right